components/Header/index.tsx

```diff
@@ -0,0 +1,31 @@
+import styled from '@emotion/styled';
+import Image from 'next/image';
+import { LoginIcon, UserIcon, LogoutIcon } from './icon';
+
+export default function Header() {
+  return (
+    <>
+      <Container>
+        <Image src="/images/logo.png" alt="로고" width={40} height={40} />
+        <ButtonContainer>
+          <LogoutIcon aria-label="로그아웃" w={10} h={10} />
+          <UserIcon aria-label="유저" w={10} h={10} />
+          <LoginIcon aria-label="로그인" w={10} h={10} />
+        </ButtonContainer>
+      </Container>
+    </>
```

Comment:
```suggestion
    <Container>
      <Image src="/images/logo.png" alt="로고" width={40} height={40} />
      <ButtonContainer>
        <LogoutIcon aria-label="로그아웃" w={10} h={10} />
        <UserIcon aria-label="유저" w={10} h={10} />
        <LoginIcon aria-label="로그인" w={10} h={10} />
      </ButtonContainer>
    </Container>
```

겉에 flagment를 주신 이유가 있으실까요? 이렇게 리팩터링 해도 동일하게 정상적으로 작동해서요!

Reply:
앗 그러내요! 기본 탬플릿을 놓고 작업하다가 수정이 덜됬나봐요 😉